src/screens/AirdropDashboard/AirdropDashboardScreen.tsx

```diff
@@ -20,26 +20,37 @@ interface Props {
   >;
 }
 
-export const AirdropDashboardScreen: FC<Props> = ({ navigation }) => (
-  <ScreenTemplate
-    footer={<Footer navigation={navigation} />}
-    header={<Header isBackArrow title={i18n.airdrop.title} />}
-  >
-    <View style={styles.wrapper}>
-      <View style={styles.infoContainer}>
-        <Text style={typography.headline4}>{i18n.airdrop.title}</Text>
-        <Text style={[styles.description, styles.spaceTop]}>{i18n.airdrop.dashboard.desc1}</Text>
-        <Text style={styles.description}>
-          {i18n.airdrop.dateOfAirdrop}&nbsp;
-          {getFormattedAirdropDate()}
-        </Text>
+export const AirdropDashboardScreen: FC<Props> = ({ navigation }) => {
+  const _isAfterAirdrop = isAfterAirdrop();
```

Comment:
Is there any reason why you don't want to call the function in JSX directly?

Reply:
I call it twice, so extracted it out here for better readability mostly